config/initializers/spree.rb

```diff
@@ -39,6 +39,9 @@
   config.s3_secret = ENV['S3_SECRET'] if ENV['S3_SECRET']
   config.use_s3 = true if ENV['S3_BUCKET']
   config.s3_protocol = ENV.fetch('S3_PROTOCOL', 'https')
+
+  # Master product price field overridden by variant price thus not required.
+  config.require_master_price = false
```

Comment:
there's no such thing as product.price. product.price is a delegate to product's  master variants price.

I'd change this explanation to: "we will create products with variants (master and standard) with no price and we will handle the consequences of that in the application"

if we let products be created without price, the products master variant will have no price and also the standard variant will have no price.

we need to test the implications of this. I think a few things will break on the shopfront if you create a product (without updating the variant price) and put it on sale...

**otherwise**, I think it is probably easier and safer to revert this line and instead just force the product price to zero on the product controller. what do you think?

Reply:
Agree on forcing the price to 0 but I guess the variant's `price` db column has a default value defined already. 

This change really needs to be thoroughly checked.